platform/mds/mds/src/main/resources/db/migration/mysql/V1__Base_version.sql

```diff
@@ -49,6 +49,16 @@ CREATE TABLE "Entity_securityMembers" (
   CONSTRAINT "Entity_securityMembers_FK1" FOREIGN KEY ("Entity_OID") REFERENCES "Entity" ("id")
 );
 
+DROP TABLE IF EXISTS "EntityDraft_fieldNameChanges";
+CREATE TABLE "EntityDraft_fieldNameChanges" (
+  "id_OID" bigint(20) NOT NULL,
+  "oldName" varchar(255) CHARACTER SET utf8 COLLATE utf8_bin NOT NULL,
+  "newName" varchar(255) CHARACTER SET utf8 COLLATE utf8_bin DEFAULT NULL,
+  PRIMARY KEY ("id_OID", "oldName"),
+  KEY "EntityDraft_fieldNameChanges_N49" ("id_OID"),
```

Comment:
I meant these auto-generated keys ending with N49, can't we change their names to something better? For example EntityDraft_fieldNameChange_KeyIdx (as far as I understand this key basically means an index on the PK?)